app/models/searchgov_domain.rb

```diff
@@ -84,6 +84,7 @@ def fetch_response
           timeout(connect: 20, read: 60).follow.get(url)
       end
     rescue StandardError => e
+      done_indexing! if activity == 'indexing'
```

Comment:
This is the right action, but upon looking at the mocked up test Martha created here (https://github.com/GSA/search-gov/compare/main...MothOnMars:search-gov:set_to_idle_1599) this just needs to be moved up into the `check_status` method on ln 48.  So something like:

```
  def check_status
    fetch_response
    if response
      record_response
    elsif activity == 'indexing'
      done_indexing!
    end
    status
  end
  ```
  
(If you want to refactor that into a separate cleaner method or tweak in whatever way, feel free.  Just proof of concept.)

I didn't dig deeply into _why_ we want `check_status` to be the trigger, but trust that Martha had a reason for this when writing up the ticket so feels safer to roll with it.

Reply:
I was trying to error out the defined domain in local machine, but the `test.gov` domain is resulting "200 OK" status all the time in local machine, but throwing an exception in staging. Need to figure that out.